.github/workflows/ci.yml

```diff
@@ -15,6 +23,14 @@ jobs:
       with:
         submodules: recursive
 
+    - uses: SymbiFlow/actions/checks@main
+      with:
+        exclude_license: |
```

Comment:
I'm pretty sure these files should have licenses?

Reply:
Yeah, I have confused them with golden generated tests, fixed.